tests/target/extern_not_explicit.rs

```diff
@@ -1,12 +1,12 @@
 // rustfmt-force_explicit_abi: false
 
-extern {
+extern "C" {
     fn some_fn() -> ();
 }
 
-extern fn sup() {}
+extern "C" fn sup() {}
 
-type funky_func = extern fn(
+type funky_func = extern "C" fn(
```

Comment:
Isn't `force_explicit_abi=false` set for this test? I wouldn't expect rustftmt to add `"C"` in this case.

Changing existing behavior like this is often a sign that the changes need to be version gated to uphold rustfmt's stability guarantee.

Reply:
Note that the source file for this test already has extern "C" and thr behavior was only changed from stripping these externs to preserve them